programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+    let answer = s.split('').sort().reverse().join('');
+    return answer;
+}
```

Comment:
answer에 할당 후에 return 하지 않고, 바로 return 해줄 수 있습미다.
```suggestion
function solution(s) {
    return s.split('').sort().reverse().join('');
}
```